pkg/controllers/cluster/cluster_controller.go

```diff
@@ -312,8 +399,9 @@ func (c *Controller) tryUpdateClusterHealth(cluster *clusterv1alpha1.Cluster) er
 			clusterHealth.status = &cluster.Status
 		} else {
 			clusterHealth = &clusterHealthData{
-				status:         &cluster.Status,
-				probeTimestamp: cluster.CreationTimestamp,
+				status:                   &cluster.Status,
```

Comment:
It could be, but now `tryUpdateClusterHealth` is a little bit complicated. I prefer to optimize this logic in the further, now maybe it's better to keep this logic as similar as possible with `k/k`🤣.